data/json/mapgen_palettes/cathedral_palette.json

```
@@ -0,0 +1,93 @@
+[
+  {
+    "type": "palette",
+    "id": "cathedral",
+    "terrain": {
+      "#": [ [ "t_region_shrub_decorative", 5 ], [ "t_region_groundcover", 1 ] ],
+      "+": "t_door_c",
+      "S": "t_linoleum_gray",
+      "T": "t_linoleum_gray",
+      "v": "t_linoleum_gray",
+      "1": "t_linoleum_gray",
+      "2": "t_linoleum_gray",
+      "3": "t_linoleum_gray",
+      "4": "t_linoleum_gray",
+      "W": "t_linoleum_gray",
+      "e": "t_linoleum_gray",
+      "I": "t_column",
+      "<": "t_stairs_up",
+      ">": "t_stairs_down",
+      "L": [ [ "t_door_locked_interior", 15 ], [ "t_door_locked", 5 ], [ "t_door_c", 2 ] ],
+      "r": "t_railing",
+      ".": "t_thconc_floor",
+      "@": "t_concrete",
+      "n": "t_concrete",
+      "s": "t_concrete",
+      "o": [ "t_window_stained_red", "t_window_stained_blue", "t_window_stained_green" ],
+      "|": "t_rock_wall",
+      "i": "t_rock_wall_half",
+      "í": "t_rock_wall_half",
+      "x": "t_wall_wood",
+      "X": "t_rock_floor",
+      "R": "t_door_metal_c",
+      "P": "t_door_bar_locked",
+      "-": "t_bars",
+      "y": "t_wall_wood"
+    },
+    "furniture": {
+      "X": [ "f_grave_monument", "f_slab", "f_grave_head", "f_grave_stone_old", "f_grave_stone" ],
+      "^": [ "f_indoor_plant", "f_indoor_plant_y" ],
+      "@": "f_statue",
+      "G": "f_statue",
+      "í": "f_statue",
+      "O": "f_organ",
+      "U": "f_bulletin",
+      "n": "f_slab",
+      "W": "f_rack",
+      "w": "f_rack_wood",
+      "B": "f_brazier",
+      "S": "f_sink",
+      "4": "f_sink",
+      "T": "f_toilet",
+      "b": "f_bench",
+      "q": "f_bench",
+      "c": "f_counter",
+      "C": "f_counter",
+      "d": "f_desk",
+      "D": "f_desk",
+      "Y": "f_rack_coat",
+      "e": "f_fridge",
+      "v": "f_oven",
+      "1": "f_cupboard",
+      "2": "f_cupboard",
+      "3": "f_cupboard",
+      "t": "f_table",
+      "h": "f_chair",
+      "H": "f_stool",
+      "l": "f_locker",
+      "a": "f_trashcan"
+    },
+    "items": {
+      "1": [ { "item": "SUS_dishes", "chance": 100 }, { "item": "SUS_silverware", "chance": 100 } ],
+      "2": { "item": "SUS_cookware", "chance": 100, "repeat": [ 1, 3 ] },
+      "3": [ { "item": "SUS_utensils", "chance": 50 }, { "item": "SUS_knife_drawer", "chance": 50 } ],
+      "4": { "item": "SUS_kitchen_sink", "chance": 100 },
+      "v": { "item": "SUS_oven", "chance": 100 },
+      "t": { "item": "silverware", "chance": 15, "repeat": [ 1, 2 ] },
+      "e": { "item": "fridge", "chance": 80, "repeat": [ 1, 5 ] },
+      "W": [
+        { "item": "SUS_pantry", "chance": 5 },
+        { "item": "big_canned_food", "chance": 10 },
+        { "item": "cannedfood", "chance": 15, "repeat": [ 1, 3 ] }
+      ],
+      "w": { "item": "cathedral_archives", "chance": 90 },
+      "Y": { "item": "coat_rack", "chance": 50, "repeat": [ 1, 5 ] },
+      "c": { "item": "makeshift_memorial", "chance": 50 },
```

```suggestion
      "c": { "item": "makeshift_memorial", "chance": 50 },
      "C": { "item": "religious_books", "chance": 25, "repeat": [ 1, 2 ] },
```
This should make it so that random items from group `religious_books` spawn on the counters over in the chancel area. Still random but religious book spawns in Cata tend to be generic anyway, plus as mentioned there isn't an exact match for what book would most likely spawn in cathedrals anyway.